include/revng/Pipeline/Kind.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+//
+#include <set>
+
+#include "revng/Pipeline/DynamicHierarchy.h"
+#include "revng/Pipeline/InvalidationEvent.h"
+#include "revng/Pipeline/Quantifier.h"
+#include "revng/Support/Assert.h"
+
+namespace Pipeline {
+class Target;
+class Context;
+
+/// The granularity tree is a tree used by targets to
+/// find out how many names are required to name a target.
```

Comment:
* [ ] Please configure your editor to properly wrap comments at 80 cols.
* [ ] Can you make a simple example?

Reply:
mh i am using our clang format to wraps things. 
shouldn't have been fixed by it?

Reply:
Weird, maybe clang-format just wraps long lines, but leaves shorter lines as they are?